frontend/javascripts/navbar.tsx

```diff
@@ -739,13 +738,9 @@ function AnonymousAvatar() {
   );
 }
 
-async function getAndTrackVersion(dontTrack: boolean = false) {
+async function getVersion() {
   const buildInfo = await getBuildInfo();
-  const { version } = buildInfo.webknossos;
-  if (dontTrack) {
-    trackVersion(version);
-  }
```

Comment:
:thinking: :upside_down_face:

Reply:
Huh, I didn’t even notice this when removing these lines :sweat_smile: